movieApp.js

```diff
@@ -1,170 +1,264 @@
+          const movieElement = createMovieCard(movie);
+          nowStreaming.appendChild(movieElement);
+        }
+      });
+    });
+
+    const login = document.getElementById("loginButton");
+    const loginPage = document.getElementById("logIn");
+
+    login.addEventListener("mouseover", function onclick() {
+      this.classList.toggle("active");
+      /* Toggle between hiding and showing the active panel */
+      if (loginPage.style.display === "block") {
+        loginPage.style.display = "none";
+      } else {
+        loginPage.style.display = "block";
+      }
+    });
+
+    const signUp = document.getElementById("signUp");
+    const signUpPage = document.getElementById("signUpPage");
+
+    signUp.addEventListener("click", function onclick() {
+      this.classList.toggle("active");
+      /* Toggle between hiding and showing the active panel */
+      if (signUpPage.style.display === "block") {
+        signUpPage.style.display = "none";
+      } else {
+        signUpPage.style.display = "block";
+      }
+    });
+  }
 }
-createMoviesPage();
-// JS 2
-const searchBtn = document.getElementById("search-icon");
-
-// function to search
-const inputField = document.getElementById("keyword-search");
-searchBtn.addEventListener("click", () => {
-  const inputStr = inputField.value.toLowerCase();
-  nowStreaming.innerHTML = "";
-  let found = false; // Flag to check if any movie is found
-
-  movies.forEach((movie) => {
-    if (
-      movie.title.toLowerCase().includes(inputStr) ||
-      movie.description.toLowerCase().includes(inputStr)
-    ) {
-      found = true;
-      const searchMovieElement = createMovieCard(movie);
-      nowStreaming.appendChild(searchMovieElement);
+// Creating screan Timer
+let timer;
+const htmlTimer = document.getElementById("timer");
+
+(function () {
+  let second = 0;
+  let minute = 0;
+  let hour = 0;
+
+  timer = setInterval(() => {
+    second++;
+    if (second === 60) {
+      second = 0;
+      minute++;
     }
-    if (!found) {
-      nowStreaming.innerHTML = `<p>Not Found</p>`;
+    if (minute === 60) {
+      minute = 0;
+      hour++;
     }
-  });
-});
-
-//
-// creating toggling in filter/shorting option
-const shorting = document.getElementById("shorting");
-const filters = document.getElementById("filters");
-shorting.addEventListener("mouseover", function () {
-  this.classList.toggle("active");
-  if (filters.style.display === "block") {
-    filters.style.display = "none";
-  } else {
-    filters.style.display = "block";
+
+    htmlTimer.innerHTML = `
+      ${hour < 10 ? "0" : ""}${hour} : 
+      ${minute < 10 ? "0" : ""}${minute} : 
+      ${second < 10 ? "0" : ""}${second}
+    `;
+  }, 1000);
+})();
+function pause() {
+  clearInterval(timer);
+}
+
+// Creating countdown
+const countdownEL = document.getElementById("countdown");
+const setTimeInMinute = 10;
+let time = setTimeInMinute * 60;
+
+const xyz = setInterval(countdown, 1000);
```

Comment:
What does xyz mean? :)

Reply:
I renamed it, that was just named while trying out the function.